### PR TITLE
Add regex validator to disallow funny characters 🥸

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

🥸 keep this guy out of our CharFields

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2262,6 +2261,10 @@ msgstr "Autre."
 msgid "Tell us the type of place."
 msgstr "De quel type de lieu s’agit-il?"
 
+#: register/forms.py
+msgid "Only enter letters or numbers."
+msgstr ""
+
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2267]: @EliseKa svp

Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:2267]:
```suggestion
#: register/forms.py
msgid "Only enter letters or numbers."
msgstr "Vous ne pouvez saisir que des lettres et des chiffres."

```

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2218,6 +2211,12 @@ msgstr "Votre session a expiré. Reconnectez-vous pour continuer à utiliser le
 msgid "— Scan a place registration"
 msgstr ""
 
+#: register/forms.py
+#, fuzzy
```
Comment (dorianjp) [inline locale/fr/LC_MESSAGES/django.po:2215]: going to need to watch this vs the fix in the other pr #545

File: register/forms.py [hunk 3]
```diff
@@ -86,10 +87,16 @@ def clean(self):
             raise forms.ValidationError(_("Tell us the type of place."))
 
 
+alphanum_validator = RegexValidator(
+    r'^[0-9A-zÀ-ÿ-_\s,.!?"\'\(\)]*$', _("Only enter letters or numbers.")
```
Comment (dorianjp) [inline register/forms.py:91]: I checked a few strings and noticed we block a fair bit of other punctuation / symbols like # or :, how open or closed do we want to be with this?

Comment (dsamojlenko) [reply register/forms.py:93]: Yeah, let's open it up to discussion. I started with alphanumeric, but added accented characters and a few random "special" chars, but not sure if we want to be more/less permissive

**Review by dorianjp — APPROVED** [review]
thanks for the update

**Pull request merged**